devtools/maven/src/main/java/io/quarkus/maven/BuildMojo.java

```diff
@@ -128,6 +128,9 @@
     @Parameter(defaultValue = "false")
     private boolean skip = false;
 
+    @Parameter(defaultValue = "false")
+    private boolean addDefaultImplementationEntries = false;
```

Comment:
This is not necessary, we are moving away from configuring the output in the mojo, and instead are moving to configuring everything in application.xml.

We are keeping the existing config for backwards compatibility but new config should not be added to the mojo.

Reply:
I removed the maven configuration option.